uniffi_bindgen/src/lib.rs

```diff
@@ -89,6 +89,9 @@
 //! to load and use the compiled rust code via its C-compatible FFI.
 //!
 
+#![warn(rust_2018_idioms)]
```

Comment:
The compiler started telling me that the `#![warn(rust_2018_idioms)]` is ignored if it's not at the crate level, so I moved those here